eventstore/test_whatsapp_actions.py

```diff
@@ -558,9 +558,9 @@ def test_handle_whatsapp_delivery_error(self, mock_get_utc_now):
                 "to": "27820001001",
                 "type": "text",
                 "text": {
-                    "body": "Sibona ukuthi imiyalezo yeWhatsApp yakho yeMomConnect "
-                    "ayifikanga. Ukuthola imiyalezo nge-SMS, phendula uthi "
-                    "\u201cSMS\u201d (kusebenza izindleko ezijwayelekile)."
+                    "body": "We see that your MomConnect WhatsApp "
+                    "messages are not being delivered. If you would "
+                    "like to receive your messages over SMS, reply ‘SMS’."
```

Comment:
These are new things, we definitely don't want to be changing these tests, although I think this functionality is currently switched off, we should keep it in case we want to switch it on again. Ensuring that the message gets translated to the user's message is important.

Reply:
makes sense, will also revert this change